dangerzone/conversion/doc_to_pixels.py

```diff
@@ -378,10 +377,11 @@ async def main() -> int:
     try:
         await converter.convert()
         error_code = 0  # Success!
-    except (RuntimeError, TimeoutError, ValueError) as e:
+    except ConversionException as e:  # Expected Errors
+        error_code = e.error_code
+    except (RuntimeError, TimeoutError, ValueError) as e:  # Unexpected Errors
```

Comment:
Should now be done in 1d2aed5d.

> Also, I think that the following line (`.update_progress`) should no longer be used, right?

These are still needed because containers version still rely this. That's how they surface the current step and progress information to the user. Once we tackle https://github.com/freedomofpress/dangerzone/issues/443 all references to `update_progress` should be removed.

Reply:
> These are still needed because containers version still rely this. That's how they surface the current step and progress information to the user. Once we tackle https://github.com/freedomofpress/dangerzone/issues/443 all references to `update_progress` should be removed.

Hm, I think you're right. In that case, should we use `.update_progress` in the `ConversionException` block as well?

Reply:
I think we want it in that case. If there is an expected error we probably don't want to should to the client the traceback.